cookbooks/convection-box/tutorial-onset-of-convection/model_input/.ipynb_checkpoints/convection-box2-old-checkpoint.prm

```diff
@@ -0,0 +1,161 @@
+# A description of convection in a 2d box. See the manual for more information.
```

Comment:
Out of curiosity, why leave old stuff around? Documentation should document the *current* state of a code base; it is not helpful to anyone to document a previous version.

Reply:
Always a good question.  The reason to keep this around is that there is not a new docker container with the most recent code.